oic.r.cred.raml

```diff
@@ -95,8 +118,19 @@ traits:
         description: The request is invalid.
 
   post:
+
```

Comment:
Uh, one problem: I don't see any schema changes for CR-50. Do they still need to be authored?